app/main.py

```diff
@@ -1,3 +1,62 @@
+from __future__ import annotations, division
```

Comment:
btw, do you use `division`?

Reply:
I had some problems passing tests with this method. After reading about __truediv__ for some reason I decided I had to use it. But now after reading about __truediv__ again, I see that the division import is used for compatibility with Python 2.